distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+* A client will only receive notifications for resources they have requested.
+* ${ddf-branding} Standard UI is subscribed to all notifications of interest to that `user/browser session: /${ddf-branding-lowercase}/notification/**`
+
+===== Publish Notifications
+
+Any application running in ${branding} can publish notifications that can be viewed by the ${ddf-branding} Standard UI or received by another notifications client.
+
+. Set a properties map containing entries for each of the parameters listed in the table above.
+. Set the OSGi event topic to `${ddf-branding-lowercase}/notification/<application-name>/<notification-type>`. Notice that there is no preceding slash on an OSGi event topic name, but there is one on the CometD channel name.
+The OSGi event topic corresponds to the CometD channel this is published on.
+. Post the notification to the OSGi event defined in the previous step.
+
+.Example for Publishing Notification
+[source,java,linenums]
+----
+Dictionary <String, Object> properties = new Hashtable<String, Object>();
+properties.put("application", "Downloads");
+properties.put("title", resourceResponse.getResource().getName());
+Long sysTimeMillis = System.currentTimeMillis();
+properties.put("message", generateMessage(status, resourceResponse.getResource().getName(), bytes, sysTimeMillis, detail));
+properties.put("user", getProperty(resourceResponse, USER));
+properties.put("status", "Completed");
+properties.put("bytes", 1024);
+properties.put("timestamp", sysTimeMillis);
+
+Event event = new Event("${ddf-branding-lowercase}/notification/catalog/downloads", properties);
+
+eventAdmin.postEvent(event);
+----
+
+==== Configuring Solr Catalog Provider Data Directory
+
+The Solr Catalog Provider writes index files to the file system.
+By default, these files are stored under `${branding}_HOME/data/solr/catalog/data`.
+If there is inadequate space in `${branding}_HOME`, or if it is desired to maintain backups of the indexes only, this directory can be changed.
+
+In order to change the Data Directory, the `system.properties` file in `${branding}_HOME/etc` must be edited prior to starting ${branding}.
+
+.Edit the `system.properties` file
+[source]
+----
+# Uncomment the following line and set it to the desired path
+#solr.catalog.data.dir=/opt/solr/catalog/data
+----
+
+==== Changing the Data Directory after ${branding} has ingested data
+
+. Shut down the ${branding}.
+. Create the new directory to hold the indexes.
++
+.Make new Data Directory
+[source,bash]
+----
+mkdir /path/to/new/data/dir
+----
++
+. Copy the indexes to the new directory.
++
+.Copy the indexes to the new Directory.
+[source,bash]
+----
+cp /path/to/old/data/dir/* /path/to/new/data/dir/.
+----
++
+. Set the `system.properties` file to use the new directory.
++
+.Set the SOLR_CATALOG_DATA_DIR
+[source,java]
+----
+solr.catalog.data.dir=/path/to/new/data/dir
+----
++
+. Restart the ${branding}.
+
+==== Configuring Thread Pools
```

Comment:
added https://codice.atlassian.net/browse/DDF-2481 for missing Windows commands.